pkg/ring/kv/consul/client.go

```diff
@@ -369,6 +369,10 @@ func (c *Client) Delete(ctx context.Context, key string) error {
 	return err
 }
 
+func (c *Client) LastUpdateTime(_ string) time.Time {
+	return time.Now().UTC()
```

Comment:
Is this true for all other KVs or we use now timestamp as a placeholder?

Reply:
It is not true, but we are just using as a placeholder. This data is more related of the stale functionally being added to ddb. As the others do not use stale data, we dont need to implement that. We can implement that later for each kv. I think we even had a issue at some point open to implement that on a ring level. I just didnt want to put more effort on that because this PR is too big already.

Reply:
Make sense!